createdb.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# TODO - it'll be trivial, but put this into a helper perhaps config.go and
+# call it any time we try to open the db and find it doesn't exist.
+# For now, run this script by hand to create the db
+db=/var/lib/lxd/lxd.db
+if [ $# -eq 1 ]; then
+	db=$1
+fi
+rm -f $db
+cat << EOF | sqlite3 $db
+create table certificates (
+id integer primary key autoincrement not null,
+fingerprint varchar(255) not null,
+type integer,
+name varchar(255) not null,
+certificate blob not null,
```

Comment:
I just noticed that I'm not being very logical here in the spec as I'm defining certificate as a PEM encoded certificate (so a string) but defined the field as BLOB.

Can you please change it to TEXT instead? I'll send an update to the spec.